bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/builders/CsvManifestsBuilder.java

```diff
@@ -162,13 +161,10 @@ private void handlePermissions(List<ClusterRole> clusterRoles, List<RoleBinding>
         Map<String, List<PolicyRule>> customPermissionRules = new HashMap<>();
         if (metadata.permissionRules != null) {
             for (CSVMetadataHolder.PermissionRule permissionRule : metadata.permissionRules) {
-                String serviceAccountName = StringUtils.defaultIfEmpty(permissionRule.serviceAccountName,
+                String serviceAccountName = defaultIfEmpty(permissionRule.serviceAccountName,
```

Comment:
Can this line be written in a single line now?

Reply:
Done.